genie-client/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     api("com.github.fge:json-patch")
     api("com.squareup.okhttp3:okhttp")
     api("com.squareup.retrofit2:retrofit")
+    api("com.squareup.retrofit2:converter-jackson")
```

Comment:
The message says 'moved', but it seems this was added

Reply:
Bah you're right I must have copied instead of cut it out cause it's still in the implementation section. Let me change that.